fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NamespaceTools.java

```diff
@@ -75,7 +75,13 @@ public static void validatePath(final Session session, final String path) {
                     try {
                         namespaceRegistry.getURI(prefix);
                     } catch (final NamespaceException e) {
-                        throw new FedoraInvalidNamespaceException("Prefix " + prefix + " has not been registered", e);
+                        if (segment.indexOf("fcr:backup") >= 0) {
+                            throw new FedoraInvalidNamespaceException(
```

Comment:
Please create a new exception type, `BackupException`, and throw that instead of a `FedoraInvalidNamespaceException`.